src/main/java/ua/softserve/rv_028/issuecitymonitor/dto/PetitionDto.java

```diff
@@ -0,0 +1,94 @@
+package ua.softserve.rv_028.issuecitymonitor.dto;
+import ua.softserve.rv_028.issuecitymonitor.entity.Petition;
+import ua.softserve.rv_028.issuecitymonitor.entity.enums.PetitionCategory;
+
+public class PetitionDto {
+
+    private long id;
+    private UserDto userDto;
+    private String title;
+    private String description;
+    private String initialDate;
+    private PetitionCategory category;
+
+    public PetitionDto() {}
+
+    public PetitionDto(Petition entity) {
```

Comment:
Dto doesn't know anything about Entity. You should use the _MapperService_ for transfer your entity to dto.

Reply:
Thank you! Done.